x-pack/plugins/infra/public/alerting/metric_threshold/components/expression_chart.test.tsx

```diff
@@ -22,6 +22,9 @@ jest.mock('../../../hooks/use_kibana', () => ({
   useKibanaContextForPlugin: () => ({
     services: {
       ...mockStartServices,
+      charts: {
```

Comment:
Please let me know if you have another idea for mocking.
I tried adding the following line to this [mock](https://github.com/elastic/kibana/blob/main/packages/core/lifecycle/core-lifecycle-browser-mocks/src/core_start.mock.ts#L26):
```
charts: chartPluginMock.createStartContract(),
```
But then it gives me another error:
```
Test suite failed to run

    TypeError: Cannot read properties of undefined (reading 'createSetup')

      11 |
      12 | const colors = new LegacyColorsService();
    > 13 | colors.init(coreMock.createSetup().uiSettings);
         |                      ^
      14 |
      15 | export const colorsServiceMock: LegacyColorsService = {
      16 |   createColorLookupFunction: jest.fn(colors.createColorLookupFunction.bind(colors)),
```